### PR TITLE
[WOE] Implement Galvanic Giant

### PR DESCRIPTION
**Pull request by xanderhall**

### CODE REVIEW

**Comment by xanderhall** [comment]
[[Galvanic Giant]]

File: Mage.Sets/src/mage/cards/g/GalvanicGiant.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+/**
+ *
+ * @author Xanderhall
+ */
+public final class GalvanicGiant extends AdventureCard {
+
+    private static FilterSpell filter = new FilterSpell("a spell with mana value 5 or greater");
+
+    {
+        filter.add(new ManaValuePredicate(ComparisonType.MORE_THAN, 4));
```
Comment (Susucre) [inline Mage.Sets/src/mage/cards/g/GalvanicGiant.java:29]: It is equivalent, but there is `new ManaValuePredicate(ComparisonType.OR_GREATER, 5)` to be closer to the rules text.

File: Mage.Sets/src/mage/cards/g/GalvanicGiant.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, new CardType[]{CardType.INSTANT}, "{3}{U}", "Storm Reading", "{5}{U}{U}");
+        
+        this.subtype.add(SubType.GIANT);
+        this.subtype.add(SubType.WIZARD);
+        this.power = new MageInt(3);
+        this.toughness = new MageInt(3);
+
+        // Whenever you cast a spell with mana value 5 or greater, tap target creature an opponent controls and put a stun counter on it.
+        Ability ability = new SpellCastControllerTriggeredAbility(new TapTargetEffect(), filter, false);
+        ability.addEffect(new AddCountersTargetEffect(CounterType.STUN.createInstance(3)).setText("and put a stun counter on it."));
```
Comment (Susucre) [inline Mage.Sets/src/mage/cards/g/GalvanicGiant.java:42]: 3 -> 1 stun counter. I think you can even use createInstance() without argument for amount=1

File: Mage.Sets/src/mage/cards/g/GalvanicGiant.java [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+/**
+ *
+ * @author Xanderhall
+ */
+public final class GalvanicGiant extends AdventureCard {
+
+    private static FilterSpell filter = new FilterSpell("a spell with mana value 5 or greater");
+
+    {
```
Comment (xenohedron) [inline Mage.Sets/src/mage/cards/g/GalvanicGiant.java:28]: need `static`

**Review by xenohedron — APPROVED** [review]

**Pull request merged**